pom.xml

```diff
@@ -203,8 +203,9 @@
                     <version>${maven.release.plugin.version}</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <preparationGoals>clean verify install</preparationGoals>
+                        <preparationGoals>clean install</preparationGoals>
                         <pushChanges>false</pushChanges>
+                        <arguments>-Prelease ${arguments}</arguments>
```

Comment:
@shaundmorris, I created https://codice.atlassian.net/browse/CAL-138 to track that work.